py/desispec/pixgroup.py

```diff
@@ -182,8 +182,16 @@ def read(cls, filename):
         night = np.tile(header['NIGHT'], nspec).astype('i4')
         expid = np.tile(header['EXPID'], nspec).astype('i4')
         tileid = np.tile(header['TILEID'], nspec).astype('i4')
+        if 'MJD-OBS' in header:
+            mjd = np.tile(header['MJD-OBS'], nspec).astype('f8')
+        elif 'MJD' in header:
+            mjd = np.tile(header['MJD'], nspec).astype('f8')
+        else:
+            mjd = np.zeros(nspec, dtype='f8')-1
```

Comment:
This 'else' version was tested and worked?

Reply:
I tested just that one line to confirm that I had my np.zeros and dtype syntax correct, but I didn't try generating files without MJD at all to confirm that it would make the MJD=-1 column.  I hope that two brains (yours and mine) are sufficient to confirm that without actually running...